docs/Getting Started/Adapting to the Head Unit Language/index.md

```diff
@@ -47,16 +47,16 @@ To customize the app name for the head unit's current language, implement the fo
 1. Set the default `language` in the !@@![iOS]`SDLLifecycleConfiguration`!@@![android,javaSE,javaEE]`Builder`.!@
 @![iOS]
 2. Add all languages your app supports to `languagesSupported` in the `SDLLifecycleConfiguration`.
-3. Implement the `SDLManagerDelegate`'s `managerShouldUpdateLifecycleToLanguage:` method. If the head unit's language is different from the default language and is a supported language, the method will be called with the head unit's current language. Return a `SDLLifecycleConfigurationUpdate` object with the new `appName` and/or `ttsName`.
+3. Implement the `SDLManagerDelegate`'s `managerShouldUpdateLifecycleToLanguage:hmiLanguage:` method. If the head unit's language is different from the default language and is a supported language, the method will be called with the head unit's current language. Return a `SDLLifecycleConfigurationUpdate` object with the new `appName` and/or `ttsName`.
 !@
 @![android,javaSE,javaEE]
-2. Implement the `sdlManagerListener`'s `managerShouldUpdateLifecycle` method. If the head unit's language is different from the default language and is a supported language, the method will be called with the head unit's current language. Return a `LifecycleConfigurationUpdate` with the new `appName` and/or `ttsName`.
+2. Implement the `sdlManagerListener`'s `//TODO` method. If the head unit's language is different from the default language and is a supported language, the method will be called with the head unit's current language. Return a `LifecycleConfigurationUpdate` with the new `appName` and/or `ttsName`.
 !@
 
 @![iOS]
 ##### Objective-C
 ```objc
-- (nullable SDLLifecycleConfigurationUpdate *)managerShouldUpdateLifecycleToLanguage:(SDLLanguage)language {
+- (nullable SDLLifecycleConfigurationUpdate *)managerShouldUpdateLifecycleToLanguage:(SDLLanguage)language hmiLanguage:(SDLLanguage)hmiLanguage {
```

Comment:
We should probably add a note that the HMI and VR language will usually be one and the same but if the OEM allows it, a user could set a UI language that is different from the VR language.